common/script/ops/scoreTask.js

```diff
@@ -87,7 +87,6 @@ function _gainMP (user, val) {
   if (user.stats.mp >= user._statsComputed.maxMP) user.stats.mp = user._statsComputed.maxMP;
   if (user.stats.mp < 0) {
     user.stats.mp = 0;
-    return user.stats.mp;
```

Comment:
I think this isn't needed - I can't see any reason why it would be. The function doesn't return anything normally, so I don't know why it would for the mp < 0 case.